src/test/fixtures/kphp_intentions/immutable_class_intention-7.fixture.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class <caret>C1 {
+    public int $field;
+}
+
+public function foo()
```

Comment:
Looks like this code is invalid:

![Image](https://github.com/user-attachments/assets/6496baa0-3761-440e-a363-36c4830f16d6)